exchange/src/client_manager/client_manager.cpp

```diff
@@ -3,58 +3,67 @@
 namespace nutc {
 namespace manager {
 
-inline bool
-ClientManager::user_exists(const std::string& uid) const
+bool
+ClientManager::user_exists(const std::string& id) const
 {
-    return clients.find(uid) != clients.end();
+    return clients.find(id) != clients.end();
```

Comment:
```suggestion
    return clients.contains(id);
```
https://en.cppreference.com/w/cpp/container/unordered_map/contains
this should probably go into the header too

Reply:
It's a private function so it's only used in the implementation file. Why put it in the header?